cfme/tests/control/test_actions.py

```diff
@@ -250,21 +190,15 @@ def vm_on(vm):
     """ Ensures that the VM is on when the control goes to the test."""
     vm.mgmt.wait_for_steady_state()
     vm.mgmt.ensure_state(VmState.RUNNING)
-    # Make sure the state is consistent
-    vm.crud.refresh_relationships(from_details=True)
-    vm.crud.wait_for_vm_state_change(desired_state=vm.crud.STATE_ON, from_details=True)
-    return vm
+    return
 
```

Comment:
Question:
- Do we need this blank ```return``` here or you missed ```vm```?

Reply:
There's no need to return `vm` with this fixture. All this fixture does is ensure that the `vm` is running.

Reply:
@john-dupuy fyi in python if not returning means we are returning "None".